docs/guides/developer/docs/release-manager.md

```diff
@@ -315,83 +315,56 @@ assume the final release should be based on `3.0-rc2`.
 
         git checkout -b r/3.0
 
-4. Add release notes, and update the changelog, then commit them:
+4. Ensure the fix versions of the relevant tickets are correct.  Search GitHub's pull request list with a query like
+   this, modifying the dates and base to match the start date of the current release, and the current date
+
+        is:pr base:r/3.x is:closed merged:2018-03-29..2018-05-31
+
+   For each of these tickets, make sure the fix version is set to the new release in JIRA, then use the
```

Comment:
I'm -1 for making release managers responsible for the accuracy of Jira tickets' metadata. For major versions, people would possibily need to go trough hundreds of tickets.

Reply:
If you really want this to happen, you could automate this and write a script which acts upon merged pull requests and updates the metadata in Jira.

Reply:
My point is that those who want to keep it should do the work and not those who already spend time on community tasks by volunteering as release managers. If it gets automated that's good. I'm not against correct metadata ;-P

As an alternative to the merge hook, you could also have the script run on the CI machines, automatically executing on builds for release branches.

Reply:
One of those is definitely what we'll do.  When I'll have the time to write the script is a good question though :)